.github/workflows/ci.yml

```diff
@@ -9,6 +9,6 @@ jobs:
             - uses: actions/checkout@master
             - uses: shivammathur/setup-php@v2
               with:
-                  php-version: 7.4
+                  php-version: 8.1
```

Comment:
we should also run tests with `8.0` since we allow it (in order to make sure that we don't use PHP enums in our codebase for instance)

```
    strategy:
        matrix:
            version: [ '8.0', '8.1' ]
        fail-fast: false
    steps:
        - uses: actions/checkout@master
        - uses: shivammathur/setup-php@v2
          with:
              php-version: ${{ matrix.version }}
```